subscription-sender/internal/queue/notification.go

```diff
@@ -0,0 +1,38 @@
+package queue
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/RinaDish/subscription-sender/internal/services"
+	"github.com/nats-io/nats.go"
+)
+
+func (queue *Queue) ConsumeSubscriptionEvent(ctx context.Context) error {
+	_, err := queue.QueueConn.Subscribe(queue.subscriptionTopicName, func(msg *nats.Msg) {
+        _ = queue.handleMessage(msg, ctx)
```

Comment:
Why did you returning error and ignoring it at the same time?

Reply:
Added error handling)